src/App.js

```diff
@@ -3,101 +3,41 @@ import './App.css';
-        let bestMove;
-        for (let i = 0; i < moves.length; ++i) {
-            game.move(moves[i]);
-            const value = minimax(depth - 1, -10000, 10000, !wantMax);
-            game.undo();
-            
-            if (value >= best) {
-                best = value;
-                bestMove = moves[i];
-            }
-        }
-
-        game.move(bestMove);
-        setFen(game.fen());
-        setGame(game);
-    };
-
-    const minimax = (depth, alpha, beta, wantMax) => {
-        if (depth === 0) return -evalBoard(game);
-
-        var moves = game.moves();
-        let best = 9999 * wantMax ? -1 : 1;
-
-        for (let i = 0; i < moves.length; ++i) {
-            game.move(moves[i]);
-            const bestChild = minimax(depth - 1, alpha, beta, !wantMax);
-            game.undo()
-
-            if (wantMax) {
-                best = Math.max(best, bestChild);
-                alpha = Math.max(alpha, best);
+    const onDrop = ({ sourceSquare, targetSquare }) => {
+      // Attempts to make the player's move.
+      const move = game.move({ from: sourceSquare, to: targetSquare });
+      if (!move) return;
+
+      return new Promise(resolve => {
+          setFen(game.fen());
+          setGame(game);
+          resolve();
+        }).then(() => {
+          setTimeout(() => {
+            if (game.game_over()) {
+              alert('Game over');
             } else {
-                best = Math.min(best, bestChild);
-                beta = Math.min(beta, best);
+              // Get the AI's best move and attempts to make the move.
+              game.move(bestMove(game, minimaxDepth));
+              setFen(game.fen());
+              setGame(game);
             }
-
-            if (beta <= alpha) return best;
-        }
-        return best
-    }
-
-    const makeMove = () => {
-        if (game.game_over()) alert('Game over');
-        minimaxRoot(minimaxDepth, true)
-    };
-
-    const evalBoard = g => {
-        let score = 0;
-        for (var i = 0; i < 8; ++i) {
-            for (var j = 0; j < 8; ++j) {
-                score += evalPiece(g.get(String.fromCharCode(i+0x61) + (8-j)), i ,j);
-            }
-        }
-        return score;
-    }
-
-    const evalPiece = (p, x, y) => {
-        if (!p) return 0;
-
-        const value = ChessWeight[p.type][p.color][y][x] + ChessWeight[p.type].modifier;
-        return p.color === "w" ? value : -value;
-    }
-
-    const onDrop = ({ sourceSquare, targetSquare }) => {
-        const move = game.move({
-            from: sourceSquare,
-            to: targetSquare,
-            promotion: "q"
-        });
-
-        if (!move) return;
-
-        return new Promise(resolve => {
-            setFen(game.fen());
-            setGame(game);
-            resolve();
-        }).then(setTimeout(makeMove, 300));
+          }, 300)
+      });
     }
 
     return (
         <div className="App">
             <Chessboard
+                draggable={!game.game_over()}
```

Comment:
check for draws again

Reply:
> [game_over](https://github.com/jhlywa/chess.js#game_over) returns true if the game has ended via checkmate, stalemate, draw, threefold repetition, or insufficient material. Otherwise, returns false.